crates/types/src/config/common.rs

```diff
@@ -625,8 +625,8 @@ pub enum MetadataClientKind {
 // TODO(azmy): Remove this Shadow struct once we no longer support
 // the `address` configuration param.
 enum MetadataClientKindShadow {
-    #[serde(alias = "embedded")]
-    Native {
+    #[serde(alias = "embedded", alias = "native")]
```

Comment:
I'm not sure we need native as an alias, it's a type that was never released. Plus, my experience with serde aliases make them surprising in some cases. I'm not sure how we'll handle collisions in this case.

Reply:
I'll clean it up. I thought I used it in the test-suite but I didn't